### PR TITLE
Fix Wi-Fi related build issues seen with upmerge branch

### PR DESCRIPTION
**Pull request by rado17**

### CODE REVIEW

File: doc/nrf/releases_and_maturity/migration/migration_guide_2.6.rst [hunk 1]
```diff
@@ -30,6 +30,16 @@ The following changes are mandatory to make your application work in the same wa
       * The ``CONFIG_CHIP_SED_ACTIVE_INTERVAL`` Kconfig option was renamed to :kconfig:option:`CONFIG_CHIP_ICD_FAST_POLLING_INTERVAL`.
       * The ``CONFIG_CHIP_SED_ACTIVE_THRESHOLD`` Kconfig option was renamed to :kconfig:option:`CONFIG_CHIP_ICD_ACTIVE_MODE_THRESHOLD`.
 
+* For samples using Wi-Fi features, following Kconfig options are not available in the current release:
```
Comment (sachinthegreen) [inline doc/nrf/releases_and_maturity/migration/migration_guide_2.6.rst]:
```suggestion
* For samples using Wi-Fi features, following Kconfig options are not available in the current release. If your application uses them they will need to be updated to remove the dependency on these:
```

File: drivers/wifi/nrf700x/Kconfig [hunk 2]
```diff
@@ -397,6 +397,13 @@ config WIFI_MGMT_SCAN_SSID_FILT_MAX
 	default 2
 endif
 
+config WIFI_MGMT_SCAN_MAX_BSS_CNT
```
Comment (sachinthegreen) [inline drivers/wifi/nrf700x/Kconfig]: This should now be `NRF_WIFI_SCAN_MAX_BSS_CNT`

File: samples/wifi/scan/Kconfig [hunk 3]
```diff
@@ -46,6 +46,20 @@ config WIFI_SCAN_CHAN_LIST
 	help
 	  Specifies the list of channels used to perform scan.
 
+config WIFI_MGMT_SCAN_DWELL_TIME_ACTIVE
```
Comment (sachinthegreen) [inline samples/wifi/scan/Kconfig]:
```suggestion
config WIFI_SCAN_DWELL_TIME_ACTIVE
```

File: samples/wifi/scan/Kconfig [hunk 4]
```diff
@@ -46,6 +46,20 @@ config WIFI_SCAN_CHAN_LIST
...
+	int "Active scan dwell time"
+	default 50
+	range 5 1000
+	help
+	  Active scan dwell time (in ms) per channel.
+
+config WIFI_MGMT_SCAN_DWELL_TIME_PASSIVE
```
Comment (sachinthegreen) [inline samples/wifi/scan/Kconfig]:
```suggestion
config WIFI_SCAN_DWELL_TIME_PASSIVE
```

**Review by sachinthegreen — CHANGES_REQUESTED** [review]

File: doc/nrf/releases_and_maturity/migration/migration_guide_2.6.rst (hunk 1 above)
Comment (rado17) [reply doc/nrf/releases_and_maturity/migration/migration_guide_2.6.rst]: Done

File: samples/wifi/scan/Kconfig (hunk 3 above)
Comment (rado17) [reply samples/wifi/scan/Kconfig]: Done

File: samples/wifi/scan/Kconfig (hunk 4 above)
Comment (rado17) [reply samples/wifi/scan/Kconfig]: Done

File: drivers/wifi/nrf700x/Kconfig (hunk 2 above)
Comment (rado17) [reply drivers/wifi/nrf700x/Kconfig]: Done

**Review by sachinthegreen — APPROVED** [review]

**Pull request merged**